docs/DocGenerator/config.yaml

```diff
@@ -0,0 +1,43 @@
+Project path: "../../tests/integration"
+
+Documentation path: ".."
+
+Include paths:
+  - wazuh_db:
+    path: "../../tests/integration/test_wazuh_db"
+    recursive: false
+  - vulnerability_detector:
+    path: "../../tests/integration/test_vulnerability_detector"
+
+Include regex:
+  - "^test_.*py$|readme.md$"
+
+Ignore paths:
+  - "../../tests/integration/test_wazuh_db/data"
+  - "*/data/*"
+
+Valid tags:
+  - DB
+  - Feeds
+  - VulDet
+  - Vulnerability Detector
+
+Required fields:
+  Module:
+    - Brief
+    - Metadata
+    - Metadata.Modules
+    - Metadata.Daemons
+    - Metadata.Operating System
+    - Metadata.Tiers
```

Comment:
Could we consider another format for this? **Brief**, **Metadata**, and **Tests** should be at the same level. 

Something like this: 
```
Required fields:
   - Brief
   - Metadata: [Modules, Daemons, Operating System, Tiers]
   - Tests: [Test Logic, Checks] 
```

Reply:
Yes. This fields will be adapted in the Sanity module development ([link](https://github.com/wazuh/wazuh-qa/issues/1616))